### ISSUE TITLE
Finetuning replication on SEED-VIG failed

### ISSUE THREAD

**Issue by zicren** [wjq-learning/CBraMod#9]
[cbramod_seedvig_ep50_bs64_lr1e-4_wd5e-2_ls0.1.log](https://github.com/user-attachments/files/19930306/cbramod_seedvig_ep50_bs64_lr1e-4_wd5e-2_ls0.1.log)

![Image](https://github.com/user-attachments/assets/0ffa6247-0679-44ba-a8c5-5e7c643f0d67)

**Comment by wjq-learning** [comment]
Hello, thank you for your interest in our work. I noticed that you might be using an older version of the code from this repository. We have made several important updates recently, including changes to the classifier design and the learning rate scheduling mechanism. Please try using the latest version of the code. You may also experiment with a learning rate of 0.0005 or set the hyperparameter `multi_lr=True` to enable the segmented learning rate strategy.

We hope this helps improve your results. Feel free to reach out if you have any further questions.
